### PR TITLE
re-factor_menu ending mechanism

### PR DESCRIPTION
**Pull request by pluAtAnsys**

### CODE REVIEW

**Comment by pluAtAnsys** [comment]
@beliaev-maksim new ending mechanism allows users to directly end game session in the score board page

File: mustards_game/score_log.py [hunk 1]
```diff
@@ -103,7 +136,10 @@ def display(self, screen):
...
                     output = 1
                     running = False
-                else:
-                    output = 0
+                elif end_button.collidepoint((mouse_x, mouse_y)):
+                    output = -1
                     running = False
+                # else:
+                #     output = 0
+                #     running = False
```
Comment (beliaev-maksim) [inline mustards_game/score_log.py:144]: ?

Comment (pluAtAnsys) [reply mustards_game/score_log.py:144]: forget to delete it...

Comment (pluAtAnsys) [reply mustards_game/score_log.py]: @beliaev-maksim done

**Review by beliaev-maksim — APPROVED** [review]

**Pull request merged**